package.xml

```diff
@@ -9,28 +10,23 @@
 
   <license>Apache-2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake_ros</buildtool_depend>
 
-  <build_depend>roscpp</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>nav_msgs</build_depend>
-  <build_depend>visualization_msgs</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>costmap_2d</build_depend>
-  <build_depend>tf</build_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>eigen</build_depend>
+  <build_depend condition="$ROS_VERSION == 1">cmake_modules</build_depend>
+  <build_depend>ros_environment</build_depend>
+  <depend condition="$ROS_VERSION == 1">costmap_2d</depend>
```

Comment:
They are also all build_depend, no? Otherwise, a line break is needed.

Reply:
`depend` is both `build_depend` as `exec_depend`

Reply:
I've added a line break to split the build_depends and the depends